content/articles/stream-filter-in-java-using-lambda-expressions/index.md

```diff
@@ -1,25 +1,29 @@
 ### Introduction
-Stream operations such as create and destroy are sandwiched by the filter() function (). In this case, it produces a Stream with elements from the input stream that meet the supplied predicate in the input stream. The filter() argument must be a stateless predicate that evaluates each component in the stream to assess whether or not it can be included.
+
+Stream operations such as create and destroy are sandwiched by the `filter()` function. In this case, it produces a `Stream` with elements from the input stream that meet the supplied predicate in the input stream. The filter() argument must be a stateless predicate that evaluates each component in the stream to assess if it can be included.
 
 ### Table of Contents

```

Comment:
Before this table of contents, I would highly recommend adding a section for prerequisites. The contents of this article don't seem very beginner-friendly so it would be overwhelming for someone newer to Java. Either that or add further explanations on things, particularly lambda expressions and maven (since you added a maven dependency in the end).